gerrit/auth.go

```diff
@@ -89,3 +116,17 @@ func (r *authManager) cleanup() {
 		r.cookiesPath_ = ""
 	}
 }
+
+func (r *credentialsAuth) GerritAuth() (gerrit.Auth, error) {
+	return gerrit.BasicAuth(r.username, r.password), nil
+}
+
+func (r *credentialsAuth) GitConfigArgs() ([]string, error) {
+	return []string{}, nil
```

Comment:
You don't need git authentication?

Reply:
Perhaps I do :) any example of git config args?

Reply:
Looking at the documentation, I don't think this would be quite that easy. Test whether you actually need this; if you do I will do some work to make it possible.

Reply:
Ok, you mean that I have to provide credentials for git separately from gerrit? I'll check it at home.

Reply:
I think that it would be something like here: https://git-scm.com/book/gr/v2/Git-Tools-Credential-Storage

Reply:
That looks promising. I'd like to do some refactoring if we need to support it.

Reply:
In git documentation: https://git-scm.com/docs/git-credential-store

The credentials storage is a file comprising entries in a format as follows:
`https://user:pass@example.com`

The file name is: .gitcredentials and the location is an user home directory. 

It seems that in a method GerritAuth I need to create this file and remove it when the Cleanup method is called. What do you think? I believe that in the afore mentioned scenario you could omit the refactoring.

Reply:
The problem is that you don't necessarily have access to the git server hostname (e.g. example.com in the above example) at that point. While it usually matches the gerrit host there are cases (that I have to deal with) where it doesn't.